contracts/Allbridge/AnteAllbridgeRugTest.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+// ┏━━━┓━━━━━┏┓━━━━━━━━━┏━━━┓━━━━━━━━━━━━━━━━━━━━━━━
+// ┃┏━┓┃━━━━┏┛┗┓━━━━━━━━┃┏━━┛━━━━━━━━━━━━━━━━━━━━━━━
+// ┃┗━┛┃┏━┓━┗┓┏┛┏━━┓━━━━┃┗━━┓┏┓┏━┓━┏━━┓━┏━┓━┏━━┓┏━━┓
+// ┃┏━┓┃┃┏┓┓━┃┃━┃┏┓┃━━━━┃┏━━┛┣┫┃┏┓┓┗━┓┃━┃┏┓┓┃┏━┛┃┏┓┃
+// ┃┃ ┃┃┃┃┃┃━┃┗┓┃┃━┫━┏┓━┃┃━━━┃┃┃┃┃┃┃┗┛┗┓┃┃┃┃┃┗━┓┃┃━┫
+// ┗┛ ┗┛┗┛┗┛━┗━┛┗━━┛━┗┛━┗┛━━━┗┛┗┛┗┛┗━━━┛┗┛┗┛┗━━┛┗━━┛
+// ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+// ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+pragma solidity ^0.8.0;
+
+import "../AnteTest.sol";
+
+/// @title Allbridge doesn't rug on mainnet test
+/// @notice Ante test to check if Allbride "rugs" or something terrible happens (at the time of deployment)
+
+contract AnteAllbridgeRugTest is AnteTest(" Allbridge mainnet bridge doesn't lose 15% of its ETH test") {
```

Comment:
Perhaps this could be renamed to a TVL plunge test?

Rugging usually implies an almost complete drop in balance / TVL.

If you want this to be a rug test. I would suggest changing the threshold to 1%.